cime_config/testdefs/testmods_dirs/clm/Meier2022_surf_rough/user_nl_clm

```diff
@@ -1,4 +1,2 @@
 z0param_method = 'Meier2022'
```

Comment:
Probably should remove this test mod since the clm4_5 and clm5_0 tests already are doing Meier off, and clm5_1 with Meier on. So both options are covered. So there's no reason for a test mod that does one or the other...

Reply:
Done